pkg/kudoctl/cmd/test.go

```diff
@@ -86,6 +87,30 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 				}
 			}
 
+			if isSet(flags, "crd-dir") {
```

Comment:
🤔 what is the benefit of this vs having `options.CRDDir` directly in `testCmd.Flags().StringVar`? If not set, you'll have the default value for that type anyway, right? I guess it at least deserves a comment since it's not obvious (at least to me)

Reply:
For consistency, mainly. It's more important below with the booleans - I want to know the difference between not set, false, and true.

Reply:
Added some comments